types/react-native-web/README.md

```diff
@@ -0,0 +1,14 @@
+# `react-native-web`
+
+> React Native for Web is an accessible implementation of React Native's Components and APIs that is interoperable with React DOM. https://necolas.github.io/react-native-web/
+Because it is not an identical implementation of `react-native`, it adds or modifies existing type signatures on exported components and APIs from `react-native`.
+
+These types are an attempt to augment existing `react-native` types to match the behavior when `react-native-web` is implemented in a project.
+
+## Usage 
+
+To enhance `react-native` types, users must import these specific types into any `.ts` or `.d.ts` file. By doing this, they are augmenting the underlying `react-native` types with web specific properties.
+
+```ts
+import 'react-native-web';
```

Comment:
You don't want to do this, probably - as far as I can tell, none of the examples in the `react-native-web` docs ever actually import the package. Instead, you want to supply `react-native-web` as a member of the `types` compiler option, or, worst-case, via ` /// <reference types="react-native-web" />` comment.

Reply:
>  none of the examples in the react-native-web docs ever actually import the package

It's true, you rarely ever import `react-native-web` in a project, this is because in most projects RNW is an alias for RN, check out [this search](https://github.com/search?q='react-native':+'react-native-web',&type=code). 

> If you want to keep these named as it is, it should actually include the runtime members of the react-native-web package in its' types (which should remove the need to suppress that last lint rule, too).

I think this is the way to go, should be fairly easy as most of the exports are similar to what RN exposes.

>  Instead, you want to supply react-native-web as a member of the types compiler option

I've never think that I could use it like this, you mean to define RNW in tsconfig's types prop like this?

```ts
{
  "compilerOptions": {
    "types": ["react-native-web"]
  }
}
```